src/components/TitleBar.js

```diff
@@ -22,6 +22,8 @@ const TitleBar = ({ id, onPrevious, title, onDismiss, titleExtras, style = {}, t
       onDismiss && h(Link, {
         'aria-label': 'Close',
         style: { marginLeft: '2rem' },
+        tabIndex: hideCloseButton ? -1 : 0,
+        'aria-disabled': !hideCloseButton,
```

Comment:
This is the opposite of what you want. When the closeButton is hidden, `aria-disabled` should be `true`.